src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Microsoft.DotNet.Build.Tasks.Feed.targets

```diff
@@ -222,6 +222,10 @@
     [Out]
     $(OrchestratedBuildId): The orchestrated build manifest's build id attribute value.
     $(OrchestratedIdentity): The human-readable full identity of the orchestrated build manifest.
+    @(OrchestratedBuild): A single item describing the root manifest element.
+      %(Identity): The name of the build.
```

Comment:
What's the difference between `$(OrchestratedIdentity)` and `@(OrchestratedBuild).Identity`?

Reply:
`OrchestratedIdentity` is more info, intended to be human-readable:

https://github.com/dotnet/buildtools/blob/c4e6ded8176e53f6bcf38f81ba7caea3cf7c7471/src/Microsoft.DotNet.VersionTools/BuildManifest/Model/BuildIdentity.cs#L59-L79

Renaming `OrchestratedIdentity` would be good because these are so similar now. I think I'll do that since this target is only used in one repo.